tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CalendarEventReplyPerformer.scala

```diff
@@ -167,12 +167,17 @@ class CalendarEventMailReplyGenerator(val bodyPartContentGenerator: CalendarRepl
         .flatMap(attachmentPart => bodyPartContentGenerator.getBasedMimeMessage(language, attendeeReply, calendarRequest)
           .map(decoratedMimeMessage => appendAttachmentPartToMail(attachmentPart, decoratedMimeMessage)))
         .map(mimeMessage => addHeaderToMimeMessage(recipient, attendeeReply.attendee, mimeMessage))
-        .map(mimeMessage => MailImpl.builder()
-          .name(generateMailName())
-          .sender(attendeeReply.attendee.asString())
-          .addRecipients(recipient)
-          .mimeMessage(mimeMessage)
-          .build()))
+        .map(mimeMessage => {
+          val mailImpl = MailImpl.builder()
+            .name(generateMailName())
+            .sender(attendeeReply.attendee.asString())
+            .addRecipients(recipient)
+            .mimeMessage(mimeMessage)
+            .build()
+          LifecycleUtil.dispose(mimeMessage)
+          mailImpl
```

Comment:
Q: Are we sure the content of the mail (mimeMessage) still can be correctly accessible after `LifecycleUtil.dispose(mimeMessage)`?

Reply:
Or we can set up the mime message without needing a copy?

`setMessageNoCopy`

Reply:
> Q: Are we sure the content of the mail (mimeMessage) still can be correctly accessible after `LifecycleUtil.dispose(mimeMessage)`?

`mimeMessage` here exactly is `MimeMessageWrapper` 
`MailImpl.builder()` wrap one more time it. 
First I also worry about it too, but I see the integration test still passed

> Or we can set up the mime message without needing a copy?

I tried it, but got another error (Distributed test)
```
S3Exception: null...
```
Error at: https://github.com/apache/james-project/blob/11646b05bbc539492fd3715230c22388174a30ad/server/blob/blob-s3/src/main/java/org/apache/james/blob/objectstorage/aws/S3BlobStoreDAO.java#L357

Reply:
:( I don't know now, The input `InputStream` was wrapped multi-layer, 
difficult to debug, `MimeMessageWrapper`, `BufferedDeferredFileOutputStream` is difficult for me

Reply:
Yes but it may be interesting to know **why**...